src/openmrs-react-root-decorator.test.js

```diff
@@ -6,19 +6,16 @@ describe("openmrs-react-root-decorator", () => {
   const opts = { featureName: "Test", throwErrorsToConsole: false };
```

Comment:
So `throwErrorsToConsole` doesn't work as a parameter to openmrsRootDecorator, it has to be a prop to the decorated component?

Reply:
Yep

Reply:
Should we get rid of it as a parameter to openmrsRootDecorator here?

Reply:
I’d be open to changing that in a future PR - might be nice to support setting it via opts. But for now that’s how it is

Reply:
My vote is we merge this PR in and then in a separate PR switch it over so that throw errors can be set via opts.

Reply:
Please do file in Jira

Reply:
https://issues.openmrs.org/browse/MF-90